README.md

```diff
@@ -181,14 +202,20 @@ The entry should look similar to: `grub,4,Free Software Foundation,grub,GRUB_UPS
 [your text here]
 
 *******************************************************************************
-### If you are re-using a previously used (CA) certificate, you will need to add the hashes of the previous GRUB2 binaries exposed to the CVEs to vendor_dbx in shim in order to prevent GRUB2 from being able to chainload those older GRUB2 binaries. If you are changing to a new (CA) certificate, this does not apply.
-### Please describe your strategy.
+### If you are re-using the CA certificate from your last shim binary, you will need to add the hashes of the previous GRUB2 binaries exposed to the CVEs mentioned earlier to vendor_dbx in shim. Please describe your strategy.
+Skip this if this is your first application for having shim signed or if you are changing to a new CA certificate.
```

Comment:
Rather than just "skip", maybe instead:

"If this is your first application or this is a new CA certificate, please say so here."